### PR TITLE
Resolves issue #240 - update submit license fields

### PR DESCRIPTION
**Pull request by goneall**
Resolves issue #240 

Added project examples:
image

The example URL's are added to the issues.  Note that the project URL's are not added to the database.

For an example, see: https://github.com/spdx/TEST-LicenseList-XML/issues/92

### CODE REVIEW

**Comment by goneall** [comment]
@swinslow @vmbrasseur Please review the wording and approach

File: src/app/forms.py
Comment (rtgdk) [inline src/app/forms.py:83]: This will cause the same issue as https://github.com/spdx/spdx-online-tools/issues/249. Since its a Char Field, so form only accept one example. In views.py
`licenseExamples = [form.cleaned_data['examples']]` will return only one example. 

We either to need to change the forms.py to accept multiple urls or send the urls as comma separated/json formatted from javascript.

Will look into it and commit the changes along with https://github.com/spdx/spdx-online-tools/issues/249 fix.

**Review by rtgdk — COMMENTED** [review]
Changes look good to me. One major change needed - same as #249

File: src/app/forms.py
Comment (goneall) [reply src/app/forms.py:83]: Agreed - any help appreciated.  I may make an attempt at a fix tomorrow based on your suggestions - I'll update the PR and issue if I'm successful.

**Comment by vmbrasseur** [comment]
1. For wording, it's "URLs", not "URL's". There's no apostrophe needed for pluralisation of an acronym.
1. The code says `exampleUrl = forms.CharField(label='Example Projects / URL', required=False)`. Should that be `required=True`? I can't picture a situation where we do _not_ want to see at least one example, ideally more.

**Comment by goneall** [comment]
@vmbrasseur Thanks for the review.

I just added commit https://github.com/spdx/spdx-online-tools/pull/248/commits/fcefdde9b439f3bacb33547875c1889f36f57d5d to make the field required and remove the apostrophe from URL's

**Comment by goneall** [comment]
> This will cause the same issue as #249. 

@rtgdk This should be resolved with commit d2f7d09

I also fixed some related issues and an issue where the form was not properly reset on submit.

This should be good to go - if you don't mind doing one more review before merging.

File: src/app/forms.py
Comment (rtgdk) [inline src/app/forms.py:83]: Should we use a TextField instead? For sourceUrl too. Although, the limit for CharField depends on the sql database we are using(255-65,535, goes to unlimited in some cases). I think we should be fine without it.
This change is not needed at all. Just pointing out for future reference.

**Review by rtgdk — APPROVED** [review]
Changes look good to me. We can merge this.

File: src/app/forms.py
Comment (goneall) [reply src/app/forms.py:83]: I couldn't find a TextField in forms - it looks like it is only in the DB model

**Pull request merged**
